docs/standards.md

```diff
@@ -8,11 +8,11 @@
 ### For all source files
 Each source code file needs to contain the license which it is licensed under.
 
-### For C / C++ / C#:
+### For C / C++ :
 
-### Variables
+#### Variables
+* structs and classes should be PascalCase; initialisms should all be the same case i.e. URL not Url.
 * static variables should start with `s_`, globals with `g_`.
-* Objects of type `struct` should be PascalCase; Abbreviations should always be uppercase (for Example favor `URL` over `url` when as a component of a named type).
```

Comment:
Seems like this was a more comprehensive and clearer version of what you replaced it with (except it was missing the fact that it should also apply to classes).